src/ui_nbgl.c

```diff
@@ -77,7 +77,7 @@ static void release_context(void)
 
 static inline uint16_t get_element_line_count(const char* line)
 {
-	uint16_t nbLines = nbgl_getTextNbLinesInWidth(BAGL_FONT_INTER_MEDIUM_32px,
+	uint16_t nbLines = nbgl_getTextNbLinesInWidth(LARGE_MEDIUM_FONT,
```

Comment:
Why the font has been modified here ?

Reply:
Because it was causing a crash on Flex as `BAGL_FONT_INTER_MEDIUM_32px` does not exist on this device.

But we have definitions `LARGE_MEDIUM_FONT` definitions for both devices (it's `BAGL_FONT_INTER_MEDIUM_32px` on Stax and `BAGL_FONT_INTER_MEDIUM_36px` on Flex)